dask_sql/datacontainer.py

```diff
@@ -212,9 +214,11 @@ def __call__(self, *args, **kwargs):
                     column_args.append(operand)
                 else:
                     scalar_args.append(operand)
+
             df = column_args[0].to_frame()
```

Comment:
```suggestion
            df = column_args[0].to_frame(self.names[0])
```

Reply:
thanks -this was masking another bug as well.